scripts/install_and_enable_addons_render.py

```diff
@@ -0,0 +1,519 @@
+"""
+To run in Blender from the terminal:
```

Comment:
Is this script description up to date with the latest changes?

If not it would be helpful to update